server/admin/http/healthstatus.go

```diff
@@ -0,0 +1,17 @@
+package adminserver
+
+type HealthStatus struct {
+	Server     string `json:"server"`
+	ValueStore string `json:"valuestore"`
+}
+
+type ValueStoreFailReason string
+
+const (
+	VS_GEN_FAILURE        ValueStoreFailReason = "Unable to generate valuestore datum"
+	VS_WRITE_FAILURE                           = "Cannot store value in valuestore"
+	VS_READ_FAILURE                            = "Cannot get value from valuestore"
+	VS_NO_DATA                                 = "Value not present in valuestore"
+	VS_READWRITE_MISMATCH                      = "Value stored does not match value retrieved from valuestore"
+	VS_REMOVE_FAILURE                          = "Unable to remove value from valuestore"
```

Comment:
Is there a reason all these consts, and their type, are exported? Are you expecting some other code to import this module to reference these?
https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps

Reply:
I was thinking of an enum that could be standardized and used elsewhere too. But if they are used only inside the indexer, makes sense to go with just messages.